src/tests/ftest/util/cart_ctl_utils.py

```diff
@@ -63,6 +63,7 @@ def __init__(self, namespace, command):
         self.l = FormattedParameter("-l {}")
         self.n = BasicParameter("-n")
         self.m = FormattedParameter("-m {}")
+        self.use_daos_agent_env = BasicParameter("--use_daos_agent_env")
```

Comment:
dash or underscore?

Reply:
Assuming `--use_daos_agent_env` is an optional command argument w/o a value (similar to something like `--debug` or, for the `daos_agent` command, `--insecure`) then it would be better implemented as:
```suggestion
        self.use_daos_agent_env = FormattedParameter("--use_daos_agent_env", False)
```
This definition makes `--use_daos_agent_env` a fixed entity and the `self.use_daos_agent_env.value` is then set to either `True` or `False` to determine if it is included as part of the command string.  Using `BasicParameter` opens us up to the possibility of someone redefining the `--use_daos_agent_env` string.